_layouts/memberpage.html

```diff
@@ -41,7 +46,7 @@ <h1 class="page-header">{{ page.name }}</h1>
 	        <!-- Contact Details Column -->
 	        <h3>Contact Details</h3>
 	        <p>
-	            {% if page.name %}<b>{{ page.name }}</b><br>{% endif %}
+	            {% if page.adressname %}<b>{{ page.adressname }}</b><br>{% else if page.title %}<b>{{ page.title }}</b><br>{% endif %}

```

Comment:
- typo aarrrg...
- It's little more complicated but I can rename 'title' to 'name'. Have to add a loop more in head.html. 
For Mehmet and for me and others? it would be better not to see four times *GmbH* or *UG (haftungsbeschränkt)*. It is more differentiation and addressname is not mandatory.